### PR TITLE
Themable startup option

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by SamuAlfageme** [comment]
@DeepDiver1975 looks like https://github.com/owncloud/client/pull/6347/commits/aecf42a46af701853a5eb31524c51cb07576d930 messed with tabs/spacing; could you check & adjust?

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 looks like aecf42a messed with tabs/spacing; could you check & adjust?

god damn - looks like my qtcreator is not properly setup? grrrr

File: src/gui/application.cpp [hunk 1]
```diff
@@ -336,7 +344,7 @@ void Application::slotownCloudWizardDone(int res)
 #ifdef Q_OS_MAC
         // Don't auto start when not being 'installed'
         shouldSetAutoStart = shouldSetAutoStart
-            && QCoreApplication::applicationDirPath().startsWith("/Applications/");
+                             && QCoreApplication::applicationDirPath().startsWith("/Applications/");
 #endif
         Utility::setLaunchOnStartup(_theme->appName(), _theme->appNameGUI(), shouldSetAutoStart);
```
Comment (guruz) [inline src/gui/application.cpp]: Did you see this invocation?

Comment (DeepDiver1975) [reply src/gui/application.cpp]: ❓ ❔ 😕

Comment (DeepDiver1975) [reply src/gui/application.cpp]: obviously not ..... crawling back into my cave

Comment (ogoffart) [reply src/gui/application.cpp]: Note that this DISABLE the auto start when setting up a second account.

Comment (DeepDiver1975) [reply src/gui/application.cpp]: So if I get this right: as soon as one account is setup auto launch is active.
why is it disabled with the second account ? THX

Comment (ogoffart) [reply src/gui/application.cpp]: maybe that's a bug.  @guruz  why do you disable auto-launch with multi account

Comment (ckamm) [reply src/gui/application.cpp]: It looks like a mistake: the commit message of fc7aaf792b26c22adaef6997bb18c58fe2c18f94 suggests it was intended to not adjust the autostart flag when further accounts are added.

**Comment by ckamm** [comment]
@DeepDiver1975 I've removed the whitespace changes and made other adjustments. Is the new `Theme` option still desired for 2.4.1? I've changed its default to `true`...

**Comment by DeepDiver1975** [comment]
@ckamm THX for taking care! The original idea to make this themable was based on my assumption that it is desired default behavior to NOT enables startup.

I was looking into an option to have influence on this default - the theme was my idea.

With #6367 I have no need for this themable change anymore. THX

**Comment by ckamm** [comment]
Great, then let's not add the option.

**Pull request closed without merging**